mozregression/main.py

```diff
@@ -255,6 +265,32 @@ def bisect_nightlies(runner, logger):
     return runner.bisect_nightlies(good_date, bad_date)
 
 
+def Preferences(prefs_files, Prefs):
```

Comment:
We usually follow the pep8 convention. Could you rename "Preferences" it to "preferences" (lowercased) ? Also "Prefs" is not good, maybe "prefs_args".